release-notes/7.0/preview/api-diff/preview6/Microsoft.WindowsDesktop.App/7.0-preview6_System.Windows.Forms.md

```diff
@@ -0,0 +1,485 @@
+-    public class DataGridViewButtonCell : DataGridViewCell {
++    [DynamicallyAccessedMembersAttribute(1)]
++    public class DataGridViewButtonCell : DataGridViewCell {
+-        public override Type EditType { get; }
++        [DynamicallyAccessedMembersAttribute(8193)]
++        public override Type EditType { get; }
+     }
+-    [ToolboxBitmapAttribute(typeof(DataGridViewButtonColumn), "DataGridViewButtonColumn")]
+-    public class DataGridViewButtonColumn : DataGridViewColumn
++    [DynamicallyAccessedMembersAttribute(1)]
++    [ToolboxBitmapAttribute(typeof(DataGridViewButtonColumn), "DataGridViewButtonColumn")]
++    public class DataGridViewButtonColumn : DataGridViewColumn
+-    [TypeConverterAttribute(typeof(DataGridViewCellConverter))]
+-    public abstract class DataGridViewCell : DataGridViewElement, ICloneable, IDisposable {
++    [DynamicallyAccessedMembersAttribute(8193)]
++    [TypeConverterAttribute(typeof(DataGridViewCellConverter))]
++    public abstract class DataGridViewCell : DataGridViewElement, ICloneable, IDisposable {
+-        [BrowsableAttribute(false)]
+-        [EditorBrowsableAttribute(2)]
+-        public virtual Type EditType { get; }
++        [BrowsableAttribute(false)]
++        [DynamicallyAccessedMembersAttribute(8193)]
++        [EditorBrowsableAttribute(2)]
++        public virtual Type EditType { get; }
+     }
+-    public class DataGridViewCheckBoxCell : DataGridViewCell, IDataGridViewEditingCell {
++    [DynamicallyAccessedMembersAttribute(1)]
++    public class DataGridViewCheckBoxCell : DataGridViewCell, IDataGridViewEditingCell {
+-        public override Type EditType { get; }
++        [DynamicallyAccessedMembersAttribute(8193)]
++        public override Type EditType { get; }
+     }
+-    [DesignTimeVisibleAttribute(false)]
+-    [DesignerAttribute("System.Windows.Forms.Design.DataGridViewColumnDesigner, System.Design, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a")]
+-    [ToolboxItemAttribute(false)]
+-    [TypeConverterAttribute(typeof(DataGridViewColumnConverter))]
+-    public class DataGridViewColumn : DataGridViewBand, IComponent, IDisposable
++    [DesignTimeVisibleAttribute(false)]
++    [DesignerAttribute("System.Windows.Forms.Design.DataGridViewColumnDesigner, System.Design, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a")]
++    [DynamicallyAccessedMembersAttribute(1)]
++    [ToolboxItemAttribute(false)]
++    [TypeConverterAttribute(typeof(DataGridViewColumnConverter))]
++    public class DataGridViewColumn : DataGridViewBand, IComponent, IDisposable
+-    public class DataGridViewComboBoxCell : DataGridViewCell {
++    [DynamicallyAccessedMembersAttribute(1)]
++    public class DataGridViewComboBoxCell : DataGridViewCell {
+-        public override Type EditType { get; }
++        [DynamicallyAccessedMembersAttribute(8193)]
++        public override Type EditType { get; }
+     }
```

Comment:
This doesn't look correct at all... 
In general, the diff looks very strange

Reply:
It's because of the detected attributes that got added. We are using a hardcoded list of attributes to ignore, but the list would have to keep growing.

Reply:
If you point me to the list I would add all the Windows Forms one there.